Wiki/Modules/FvwmPager/index.md

```diff
@@ -18,3 +18,71 @@ active windows are. Windows in the pager are shown in the same color as
 their fvwm decorations.  The pager can be used to change your viewport into
 the current desktop, to change desktops, or to move windows around.
 
+## Configuration and Use
+
+FvwmPager is used to show the layout of your desktop. It will show all the Pages
+and Desktops you have set up, and the windows opened in each.
+
+A single fvwm Desktop is split up into a grid of pages. The number of pages is
+configured with the DesktopSize command. For the FvwmPager configuration, refer
+to the man page for a full explanation of all the options available. This
+configuration is the same as in fvwm default config.
+
+![image](pager2x2.png){: style="float: right; margin-right: 5em; margin-top: 1em;"}
+
+{% highlight fvwm %}
+{% capture txt %}
+DesktopSize 2x2
+
+DestroyModuleConfig FvwmPager: *
+*FvwmPager: Geometry 150x150-0+0
+*FvwmPager: Font None
+*FvwmPager: Colorset * 10
+*FvwmPager: HilightColorset * 13
+*FvwmPager: BalloonColorset * 10
+*FvwmPager: WindowColorsets 10 11
+*FvwmPager: Balloons All
+*FvwmPager: BalloonFont "xft:Sans:Bold:size=8:antialias=True"
+*FvwmPager: BallonYOffset +2
+*FvwmPager: Window3dBorders
+*FvwmPager: MiniIcons"
+*FvwmPager: UseSkipList
+{% endcapture %}
+{{ txt }}
+{% endhighlight %}
+
+Next is to set up the Styles and Colorsets you want to use for your pager.
+
+{% highlight fvwm %}
+Style "FvwmPager" NoTitle, !Handles, !Borders, Sticky, WindowListSkip, \
+ CirculateSkip, StaysOnBottom, FixedPosition, FixedSize, !Iconifiable
+{% endhighlight %}
+
+To launch the FvwmPager, you run it as <strong>Module FvwmPager * </strong>
```

Comment:
Don't use `<strong>` for code, for code you should use code blocks, which are back ticks, so instead use `` `Module FvwmPager *` ``.

Reply:
Also you should never have to write html code (except in special cases), if you wanted bold you should have done `**Module FvwmPager \***` or `__Module FvwmPager \*__`, note the extra backslash, `\*`, is to escape or tell markdown to not use this for a start formatting block.